<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppletEditAboutMeUi</class>
 <widget class="QWidget" name="AppletEditAboutMeUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>703</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit About Me</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="VxPushButton" name="m_ViewAboutMePageButton">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_ViewServiceLabel">
       <property name="text">
        <string>View My About Me Page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PermissionWidget" name="m_PermissionWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_AboutMeGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_ApplyAboutMeButton">
        <property name="text">
         <string>Apply About Me Changes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_GreetingLabel">
        <property name="text">
         <string>Greeting:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_GreetingEdit">
        <property name="text">
         <string>Hi There!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aboutMeLabel">
        <property name="text">
         <string>About Me:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="m_AboutMeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="plainText">
         <string>I'm Wonderfull!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_PictureOfMeLabel">
        <property name="text">
         <string>My Picture</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="m_BrowsePictureButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_TakeSnapshotButton">
          <property name="text">
           <string>Take Snapshot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_PictureOfMeFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="favoriteWebsitesLabel">
        <property name="text">
         <string>My Favorite Websites</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_FavoriteWebsite1Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="m_FavoriteWebsite2Edit">
        <property name="text">
         <string>https://donate.doctorswithoutborders.org</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_FavoriteWebsite3Edit">
        <property name="text">
         <string>http://www.nolimitconnect.org</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Donation Information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="m_DonationEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VxPushButton</class>
   <extends>QPushButton</extends>
   <header>VxPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>PermissionWidget</class>
   <extends>QWidget</extends>
   <header>PermissionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
